docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -553,6 +553,19 @@ to the relative name of the config. For example,
 ``custom_runtime_config: GREATCONFIG.conf``.
 
 
+``driver_tar_uri``
+"""""""""""""""""""""""""""""
+
+This is an advanced feature - under normal conditions, omit this key.
+When this key is present, the local driver will not build from source.
+Instead, during the `firesim infrasetup`, this `.tar.gz` file will be
+deployed and extracted into the `sim_slot_X` folder. The purpose of this
+feature is to enable advanced CI configurations where the driver build step
```

Comment:
I think you can sell this feature a little harder, because it's cool and very useful: 

This feature allows sharing simulator drivers alongside their bitstreams (AGFIs or XCLBINs) in a HWDB entry. This eases sharing simulators with other users by avoiding both long rebuilds of the driver and potential driver-bitstream incompatibilities.

Reply:
I would argue that the purpose of this is what David said.  The CI implications are nice but I've wanted this for a long time because it is both onerous and fraught to have to rebuild the driver at runtime.